lib/template.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * replaces all uppercase dollar templates with their string representation from obj
+ * if replacement is undefined in obj the dollar template string is left untouched
+ */
+
+const template = (string, obj) => {
+  return string.replace(/(\$[A-Z_]+)/g, (_, k) =>
+    obj[k] === undefined ? k : `${obj[k]}`
+  )
+}
+
+module.exports.template = template
+
+String.prototype.template = function(obj) {
```

Comment:
Is there a reason you're overwriting the prototype here? 🤔

Reply:
Overwrite? 🤔
 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/prototype

I am adding template function to String. I'd guess it is a style choice. I can remove it 😊

Reply:
Yeah, let’s stick as close to simple functions if we can.